openpype/hosts/maya/plugins/publish/extract_camera_mayaScene.py

```diff
@@ -152,22 +161,23 @@ def process(self, instance):
                 with lib.suspended_refresh():
                     if bake_to_worldspace:
                         self.log.info(
-                            "Performing camera bakes: {}".format(transform))
+                            "Performing camera bakes: {}".format(transforms))
                         baked = lib.bake_to_world_space(
-                            transform,
+                            transforms,
                             frame_range=[start, end],
-                            step=step
+                            step=step,
+                            copy_input_connections=self.keep_input_connections
```

Comment:
`self.keep_input_connections` always seem to get its value from settings rather than from the instance.